platform/android/MapLibreAndroidTestApp/src/main/AndroidManifest.xml

```diff
@@ -874,6 +874,18 @@
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value=".activity.FeatureOverviewActivity" />
+        </activity> <!-- Telemetry -->
```

Comment:
Could you name this something else? I don't think it is telemetry (because the data is not sent anywhere) and because users often ask if we have any telemetry. I don't want there to be any confusion.

Reply:
Renamed to "Events"